packages/ui-components/Common/ChangeHistory/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { ChevronDownIcon, ClockIcon } from '@heroicons/react/24/outline';
+import type { FC } from 'react';
+
+import styles from './index.module.css';
+
+type ChangeHistoryProps = React.JSX.IntrinsicElements['div'] & {
+  label: string;
+  changes: Array<{
+    versions: Array<string>;
+    label: string;
+    url?: string;
+  }>;
+  align?: 'left' | 'right';
+};
+
+const ChangeHistory: FC<ChangeHistoryProps> = ({
+  label = 'History',
+  changes = [],
+  align = 'right',
+  className,
+  'aria-label': ariaLabel = label,
```

Comment:
Will these ever need to be different than the label?

Reply:
I was wondering in case you want to say "Change History" in the aria label

Reply:
Yep, but it's being applied to `dropdownContentWrapper` so it's not where `{ ...props }` are being used